apps/etterlatte-egne-ansatte-lytter/src/test/kotlin/SkjermingslesingTest.kt

```diff
@@ -0,0 +1,97 @@
+
+import io.confluent.kafka.serializers.KafkaAvroDeserializerConfig
+import io.confluent.kafka.serializers.KafkaAvroSerializer
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.runs
+import io.mockk.verify
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.cancelAndJoin
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import no.nav.common.KafkaEnvironment
+import no.nav.etterlatte.BehandlingKlient
+import no.nav.etterlatte.kafka.KafkaConsumerConfiguration
+import no.nav.etterlatte.kafka.KafkaConsumerEgneAnsatte
+import org.apache.kafka.clients.CommonClientConfigs
+import org.apache.kafka.clients.consumer.ConsumerConfig
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerConfig
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.common.serialization.StringDeserializer
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import java.util.*
+
+@TestInstance(TestInstance.Lifecycle.PER_METHOD)
+class SkjermingslesingTest {
+
+    companion object {
+        val pdlPersonTopic = "nom.skjermede-personer-status-v1"
+        val kafkaEnv = KafkaEnvironment(
+            noOfBrokers = 1,
+            topicNames = listOf(pdlPersonTopic),
+            withSecurity = false,
+            autoStart = true,
+            withSchemaRegistry = true
+        )
+    }
+
+    @Test
+    fun testSkjermingshendelse() {
```

Comment:
savner kanskje et litt mer beskrivende navn for hva som skjer her
`skal oppdatere sak i behandling naar vi mottar skjermingshendelse` eller noe sånt?

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/884/commits/56c0c9f61944604e11e32f1f907e051ea1cbb3be